branching.js

```diff
@@ -2,6 +2,8 @@ M.profile_field_branching = {};
 
 M.profile_field_branching.init = function(Y, fieldid, parent1id, desired1, parent2id, desired2) {
 
+    var logging = false;
```

Comment:
Maybe set this to M.cfg.developerdebug ?

Reply:
Thanks for the tip with YUI. Updated.